asm/canonical-service/controller.yaml

```diff
@@ -14,7 +14,6 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.2.5
     gke.io/cluster: "gke://PROJECT_ID/us-central1-c/asm-cluster" # kpt-set: gke://${gcloud.core.project}/${gcloud.compute.location}/${gcloud.container.cluster}
-  creationTimestamp: "null"
```

Comment:
Why remove this ? the fix is not released yet but you can retain this field without quotes ?